core/jobs.py

```diff
@@ -103,7 +103,13 @@ def _is_abstract(cls):
 
     @classmethod
     def create_new(cls):
-        """Creates a new job of this class type. Returns the id of this job."""
+        """Creates a new job of this class type.
+        Returns:
```

Comment:
Here and below, add blank line above this one. (See other files in #2394 for the expected format.)